aviatrix_ha.py

```diff
@@ -504,6 +506,25 @@ def verify_backup_file(controller_instanceobj):
         return True, s3_file
 
 
+def support_upgrade_to_build(ip_addr, cid):
```

Comment:
You could rename to `is_upgrade_to_build_supported`

Reply:
Changed